### PR TITLE
[RHCLOUD-15003] Rename indexes and foreign keys

### PR DESCRIPTION
**Pull request by MikelAlejoBR**
Adds a migration to rename all the keys and foreign keys to mimic the default names that PosgreSql uses when it creates new indexes or foreign keys and no custom name is provided for them.

**NOTE:** https://github.com/RedHatInsights/sources-api-go/pull/174 needs to be merged first, and then, this PR needs to be rebased.

## Links

https://issues.redhat.com/browse/RHCLOUD-15003

### CODE REVIEW

**Review by lpichler — APPROVED** [review]
:clap: LGTM !

**Review by petracihalova — APPROVED** [review]

**Comment by lindgrenj6** [comment]
Adding a label here - `wait-to-merge` since we aren't going to merge these until we un-deploy the rails app.

**Comment by lpichler** [comment]
This needs rebase.

**Comment by MikelAlejoBR** [comment]
Rebased!

**Comment by MikelAlejoBR** [comment]
&

**Comment by lpichler** [comment]
Now it could be rebased 🙏 thanks!

**Comment by MikelAlejoBR** [comment]
> Now it could be rebased pray thanks!

Sure, just pushed again!

**Comment by lpichler** [comment]
great let's wait for @lindgrenj6 approval.

**Review by lindgrenj6 — APPROVED** [review]

**Pull request merged**
